_posts/2018-08-01-umbrellas-just-when-it-rains.md

```diff
@@ -0,0 +1,84 @@
+---
+author: Sean Callan
+author_link: https://github.com/doomspork
+categories: general
+date: 2018-08-01
+ayout: post
+title: Umbrellas: only when it rains?
+excerpt: >
+  A look at Umbrella applications and how they can help us write cleaner maintainable code.
+---
+
+The Umbrella application is powerful tool available to us but many people are unfamiliar with them.
+In this post we'll look at Umbrella applications and why we might want to consider using them for our next project.
+
+## Project types
+
+Before we can understand when to use Umbrellas and the role they play, let's review the types of projects available.
+For all intents and purposes there are three project types in the Erlang and Elixir ecosystem: Applications, Libraries, and Umbrellas.
+
+Libraries are packages of code that can be re-used by other projects and applications.
+These projects lack a supervision tree.
```

Comment:
Maybe add this line to the one below, like this:
A Library would lack the `:mod` key in the Mix `application/0` function and also would lack a supervision tree.